grouper/graph.py

```diff
@@ -60,46 +60,72 @@ class NoSuchGroup(Exception):
 class GroupGraph(object):
     """The cached permission graph.
 
+    The graph is internally represented by four major components: the users, groups, and
```

Comment:
Nit: Maybe note that none of these components include disabled anything? In general I think getting more explicit about how we handle disabled things is a good idea, since right now I have very low confidence that we're getting it all right

Reply:
Good idea. Added some additional commentary about that.